gremlin-go/driver/graphTraversal.go

```diff
@@ -669,6 +670,30 @@ func (g *GraphTraversal) Write(args ...interface{}) *GraphTraversal {
 	return g
 }
 
+func int32Args(args ...interface{}) []interface{} {
+	for i, arg := range args {
+		switch val := arg.(type) {
+		case uint8:
```

Comment:
`uint8`, `int8`, `int16` are not needed. Times() accept numbers shorter than integers.